pkg/ocm/config.go

```diff
@@ -91,6 +91,13 @@ func GetEnv() (string, error) {
 	urlAliases := URLAliases
 	if cfg.FedRAMP {
 		urlAliases = fedramp.URLAliases
+		// Special use case for Admin users in the GovCloud environment
```

Comment:
I think this is not required if we are adjusting the urlAliases variable. We can always resort for the fallback as they are the same code block if it doesn't include openshift. What do you think?